data/json/monsters/zed_misc.json

```diff
@@ -479,6 +479,66 @@
     ],
     "armor": { "bash": 8, "cut": 5, "bullet": 4, "electric": 2 }
   },
+  {
+    "id": "mon_zombie_grue",
+    "type": "MONSTER",
+    "name": { "str": "zombie prowler" },
+    "description": "Enveloped in darkness is the outline of what used to be a person, quick and spindly - and nigh invisible in the shadows.  Even with the unnatural camouflage hiding its features you can make out large teeth and even larger claws.",
+    "default_faction": "zombie",
+    "looks_like": "mon_zombie_brute_ninja",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 110,
```

Comment:
"30 HP more than a fresh zed" is very much below par for a late zomboni. For reference, a point blank unaimed M4 shot takes 0,5s and does ~40 on a normal hit at something like 20% to-hit on a halfway competent character, so they survive 7 seconds *if* they get the drop on ranged toon (not to mention a late beefy melee character). I'd go with 160 at least. *Alternatively* keep the lowish (but slightly higher) HP and add regen in the darkness to double up on the ninjaness.